src/main/resources/application.yaml

```diff
@@ -35,3 +35,11 @@ queue:
   notifications:
     read-connection-string: ${QUEUE_NOTIFICATIONS_READ}
     max-delivery-count: ${QUEUE_NOTIFICATIONS_MAX_RETRY}
+
+# clients region
+
+idam:
```

Comment:
idam ?

Reply:
isn't it idam endpoint?

Reply:
i think we using such config tree everywhere

Reply:
nopes s2s and idam are two different entities

Reply:
should it be s2s-auth.url instead of idam.s2s-auth.url ?

Reply:
this is still not addressed

Reply:
:boom: